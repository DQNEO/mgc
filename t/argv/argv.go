package main

import (
	"os"
)

func main() {
	fmtPrintf("%d\n", len(os.Args))
	fmtPrintf("%s\n", bytes(os.Args[1]))
}
