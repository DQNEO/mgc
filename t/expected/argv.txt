2
/etc/resolv.conf
